install_config/storage_examples/shared_storage.adoc

```diff
@@ -13,4 +13,473 @@ toc::[]
 == Overview
 The following use case describes how a user wanting to leverage shared storage for use by two seperate containers would configure the solution. This example highlights the use of NFS, but can easily be adapted to other shared storage types, such as GlusterFS. In addition, this example will show configuration of pod security as it relates to shared storage.
 
+For an explanation of persistent volumes (PVs), persistent volume claims (PVCs), and
+using NFS as persistent storage, please see the section on
+link:../persistent_storage/persistent_storage_nfsfs[Persistent Storage Using NFS].
+Below, is an end-to-end example of using an existing nfs cluster as an OpenShift persistent store.
+It is assumed that a working NFS server and exports  has already been setup. 
+
+= Sharing storage across two containers using NFS
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+
+[NOTE]
+====
+All `oc ...` commands are executed on the OpenShift master node.
+====
+
+=== 1. Create the Persistent Volume
+The persistent volume (PV) file is defined next before creating the PV object in
+OpenShift:
+
+.Persistent Volume Object Definition Using NFS
+====
+
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: nfs-pv <1>
+spec:
+  capacity:
+    storage: 1Gi <2>
+  accessModes:
+    - ReadWriteMany <3>
+  persistentVolumeReclaimPolicy: Retain <4>
+  nfs: <5>
+    path: /opt/nfs <6>
+    server: nfs.f22 <7> 
+    readOnly: false
+----
+<1> The name of the PV which is referenced in pod definitions or displayed in
+various `oc` volume commands.
+<2> The amount of storage allocated to this volume.
+<3> `accessModes` are used as labels to match a PV and a PVC. They currently
+do not define any form of access control. 
+<4> A volume reclaim policy of "retain" indicates to preserve the volume after the pods 
+<5> This defines the volume type being used, in this case the *nfs* plug-in.
+<6> This is the NFS mount path
+<7> This is the NFS server. This can also be specified by IP address.
+====
+
+Save the PV definition to a file, for example *_nfs-pv.yaml_*,
+and create the persistent volume:
+
+====
+----
+# oc create -f nfs-pv.yaml 
+persistentvolume "nfs-pv" created
+----
+====
+
+Verify that the persistent volume was created:
+
+====
+----
+# oc get pv
+NAME         LABELS    CAPACITY   ACCESSMODES   STATUS      CLAIM     REASON    AGE
+nfs-pv   <none>    1Gi        RWX           Available                       37s
```

Comment:
Did this output line get pasted correctly? It does not align with the column headers. `oc get pv`, for me, has the output columns aligned under the header columns...